### ISSUE TITLE
Request/Suggestion - commands and perms

### ISSUE THREAD

**Issue by mouro001** [the-obsidian/DiscordBridge#21]
Sorry I think this is on the wrong place, but I don't know how to work with github.
Is possible to have some command to talk to discord? And perms to see the messages? Like as a member /staff doesn't work and I can't see staff chat, but with Moderator /staff (message) post the message do discord and I can see discord chat.

**Comment by jacobwgillespie** [comment]
Nope, you've got the right place.  So, you're essentially wanting a command that posts messages to Discord rather than having all chat messages go to discord?  And a permission node to enable access to that command?  That makes some sense - not sure if it's within the scope of this plugin or not though.

**Comment by mouro001** [comment]
Yes, to use it like a staff chat, that only staff can see and send messages. and on config have a option to select commands and permissions or all.
